js/piwik.js

```diff
@@ -3561,6 +3568,10 @@ if (typeof window.Piwik !== 'object') {
                     return false;
                 }
 
+                if (success && typeof callback === 'function') {
+                    callback({request: request, trackerUrl: configTrackerUrl, success: true});
```

Comment:
Since this assumes success, would it help to add, eg, `isSendBeacon: true`? Just in case someone might want to tell. I'm not sure of a use case for that though.

Reply:
doesn't hurt 👍  will add it

Reply:
done